pyaedt/emit.py

```diff
@@ -45,44 +271,50 @@ class Emit(FieldAnalysisEmit, object):
         Whether to release AEDT on exit. The default is ``False``.
     student_version : bool, optional
         Whether to open the AEDT student version. The default is ``False``.
+    port : int, optional
+        Port number on which to start the oDesktop communication on an already existing server.
+        This parameter is ignored when creating a new server. It works only in 2022 R2 or later.
+        The remote server must be up and running with the command `"ansysedt.exe -grpcsrv portnum"`.
     machine : str, optional
         Machine name to which the oDesktop session is to connect to. This
         parameter works only in 2022 R2 and later. The remote server must be
         up and running with the command `"ansysedt.exe -grpcsrv portnum"`.
         If the machine is `"localhost"`, the server starts if it is not present.
```

Comment:
What does "it" refer to in "if it is not present"?

Reply:
It refers to this parameter